drop-storage/src/lib.rs

```diff
@@ -976,16 +973,9 @@ impl Storage {
         Ok(out)
     }
 
-    fn get_outgoing_paths(&self, transfer_id: Uuid) -> Result<Vec<OutgoingPath>> {
+    fn get_outgoing_paths(conn: &Connection, transfer_id: Uuid) -> Result<Vec<OutgoingPath>> {
         let tid = transfer_id.to_string();
 
-        trace!(
-            self.logger,
-            "Fetching outgoing paths for transfer";
-            "transfer_id" => &tid
-        );
-
-        let conn = self.pool.get()?;
```

Comment:
Why are trace! calls gone?

Reply:
It's a free function now, does not have access to `logger` anymore

Reply:
I can change it to take a logger as an argument. What do you think?

Reply:
I think that more logging is helpful when debugging things but you decide if it's worth it to have it here :)